README.md

```diff
@@ -125,6 +125,20 @@ cd torch2trt
 sudo python setup.py install --plugins
 ```
 
+### Option 3 - With experimental support for Quantization Aware Training
+
+To install with `torch2trt_contrib` to play with QAT workflow that is supported in `TensorRT>=7.0` , call the following
+
+```bash
+git clone https://github.com/NVIDIA-AI-IOT/torch2trt
+cd torch2trt
+sudo python setup.py install --plugins --experimental
```

Comment:
--experimental to --contrib.

Reply:
### Option 3 - With support for 'contrib' functionality and examples

To install torch2trt with ``torch2trt.contrib``, which includes experimental community contributed functionality like Quantization Aware Traning, call the following

```bash
sudo python setup.py install --contrib
```
Contributed components might have additional dependencies.  Please see ``examples/contrib`` and ``torch2trt/contrib`` for more information on these packages.

Reply:
Done!